EXILED/Exiled.Events/Patches/Events/Player/ChangingRoleAndSpawned.cs

```diff
@@ -152,8 +147,18 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
 
                     // ChangingRole.ChangeInventory(changingRoleEventArgs, oldRoleType);
                     new(OpCodes.Call, Method(typeof(ChangingRoleAndSpawned), nameof(ChangeInventory))),
+                });
+
+            newInstructions.InsertRange(
+                newInstructions.Count - 1,
+                new CodeInstruction[]
+                {
+                    // if (this.isLocalPlayer)
+                    //     return;
+                    new(OpCodes.Ldarg_0),
+                    new(OpCodes.Callvirt, PropertyGetter(typeof(PlayerRoleManager), nameof(PlayerRoleManager.isLocalPlayer))),
+                    new(OpCodes.Brtrue_S, returnLabel),
```

Comment:
plus, i want to discuss about that change, i merge `ChangeInventory` call and `OnSpawned` events for not call `spawned` when player role set to `None`, you have a reason to revert that?

Reply:
Because it was broken.

I'm unsure what part of your change caused it, or if it was a mixture of weird merge artifacts, but when I merged your changes for this patch, it broken inventory assignment when spawning as any role, so I reverted it back to the modified version of the original patch from master that I had made changes to simply get it working.